tests/vspec/test_static_uids/test_static_uids.py

```diff
@@ -27,14 +27,24 @@
 # HELPERS
 
 
-def get_cla_test(test_file: str):
-    return (
-        "../../../vspec2id.py "
-        + test_file
-        + " ./out.vspec --validate-static-uid "
-        + "./validation_vspecs/validation.vspec "
-        + "--only-validate-no-export"
-    )
+def get_cla_test(test_file: str, overlay: str | None = None):
+    if overlay:
+        return (
+            "../../../vspec2id.py "
+            + test_file
+            + " -o "
+            + overlay
+            + " ./out.vspec --validate-static-uid "
+            + "./validation_vspecs/validation.vspec "
+        )
+    else:
+        return (
+            "../../../vspec2id.py "
+            + test_file
+            + " ./out.vspec --validate-static-uid "
+            + "./validation_vspecs/validation.vspec "
+            + "--only-validate-no-export"
```

Comment:
To me it is not obvious why you have `"--only-validate-no-export"` here but not for the overlay alternative. Possibly add a comment.